ngraph/frontend/paddlepaddle/src/op/cast.cpp

```diff
@@ -0,0 +1,33 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <ngraph/opsets/opset6.hpp>
+#include "cast.hpp"
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+
+OutputVector cast (const NodeContext& node) {
+    auto data = node.get_ng_input("X");
+    auto out_dtype = node.get_attribute<int>("out_dtype");
+    auto dtype = node.get_attribute<ngraph::element::Type>(std::to_string(out_dtype));
```

Comment:
I think this code is incorrect. You should just write `node.get_attribute<ngraph::element::Type>("out_dtype")`. `out_dtype` is represented as integer but writing `get_attribute<ngraph::element::Type>` we are requesting conversion of this integer to ngraph's type.

In the code here I see that we are querying attribute twice: `out_dtype` and then, something strange, we represent integer as a string and trying to get such attribute again. But there are no such attributes in `cast` node with names as integer numbers. There is only `out_dtype` related to that.